plugins/woocommerce/src/Internal/Admin/Logging/PageController.php

```diff
@@ -305,28 +463,67 @@ private function handle_list_table_bulk_actions(): void {
 			wp_safe_redirect( remove_query_arg( $removable_args, $request_uri ) );
 			exit;
 		}
+	}
 
-		$deleted = filter_input( INPUT_GET, 'deleted', FILTER_VALIDATE_INT );
+	/**
+	 * Format a log file line.
+	 *
+	 * @param string $text        The unformatted log file line.
+	 * @param int    $line_number The line number.
+	 *
+	 * @return string
+	 */
+	private function format_line( string $text, int $line_number ): string {
+		$classes = array( 'line' );
 
-		if ( is_numeric( $deleted ) ) {
-			add_action(
-				'admin_notices',
-				function() use ( $deleted ) {
-					?>
-					<div class="notice notice-info is-dismissible">
-						<p>
-							<?php
-							printf(
-								// translators: %s is a number of files.
-								esc_html( _n( '%s log file deleted.', '%s log files deleted.', $deleted, 'woocommerce' ) ),
-								esc_html( number_format_i18n( $deleted ) )
-							);
-							?>
-						</p>
-					</div>
-					<?php
-				}
+		$level_severities = range( 100, 800, 100 );
+		$severity_levels  = array();
+		foreach ( $level_severities as $severity ) {
+			$severity_levels[] = WC_Log_Levels::get_severity_level( $severity );
+		}
```

Comment:
Not essential we change this, but rather than figure this out here, what are your thoughts on adding a new method to `WC_Log_Levels` such as `get_severity_levels()` (plural), that basically returns (crurently protected property) `WC_Log_Levels::$severity_to_level`?

Reply:
That's a great idea ﻿5fee9226ae